### PR TITLE
Add option to template directory

### PR DESCRIPTION
**Pull request by radhus**
Haven't tested it with actual vault yet 🙄

### CODE REVIEW

File: cmd/vaultify/vaultify.go [hunk 1]
```diff
@@ -148,6 +148,8 @@ func init() {
 	templateCmd.Flags().StringVar(&flags.templateOptions.Role, "role", "", "Vault kubernetes role to assume")
 	templateCmd.Flags().StringVar(&flags.templateOptions.TemplateFileName, "template-file", "", "Template file to render")
 	templateCmd.Flags().StringVar(&flags.templateOptions.OutputFileName, "output-file", "", "Output file")
+	templateCmd.Flags().StringVar(&flags.templateOptions.TemplateDirName, "template-dir", "", "Template diretory to render all files in")
```
Comment (ahilsend) [inline cmd/vaultify/vaultify.go]:
```suggestion
	templateCmd.Flags().StringVar(&flags.templateOptions.TemplateDirName, "template-path", "", "Template diretory to render all files in")
```

How about we have a generic flag. And based on whether the path is a file or directory we choose what to do?

In that case we could mark the old flags as deprecated :)

**Review by ahilsend — DISMISSED** [review]
Looks very nice! Thanks! :blush: 

I would suggest a small change to the CLI flags, rest looks good. And we need to test it.

On a side note, would be convenient to allow different auth modes, then we could simplify testing via docker-compose :thinking:

File: cmd/vaultify/vaultify.go (hunk 1 above)
Comment (radhus) [reply cmd/vaultify/vaultify.go]: I had the same thought, but was afraid of deprecating. But keeping the old and just mark them deprecated could work. I'll do that!

**Comment by ahilsend** [comment]
Added 2 commits, hope that's fine with you? Tested this locally, and it works

**Comment by radhus** [comment]
Looks good to me, merge whenever you want 👏

**Review by ahilsend — APPROVED** [review]

**Pull request merged**